media/themes/modern.msstyles/lang/zh-CN.rc

```diff
@@ -0,0 +1,42 @@
+/* Simplified Chinese translation by Henry Tang Ih 2015 (henrytang2@hotmail.com) */
+/* Improved by Luo Yufan (njlyf2011@hotmail.com) */
+
+LANGUAGE LANG_CHINESE, SUBLANG_CHINESE_SIMPLIFIED
+
+STRINGTABLE
+{
+1000, "亚白"
+1001, "黑暗"
+}
+
+STRINGTABLE
+{
+2000, "???"
+2001, "???"
```

Comment:
Something broken here?

Reply:
i dont speak chinese so i cant translate it

Reply:
In that case please just enter the english text